content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+</Callout>
+
+<Card href="https://multipaper.io/download.html" title="Download">
+  Download a jarfile for MultiPaper here.
+</Card>
+
+## 1.8 Softwares
+There are some spigot or paper forks that are actively maintained for 1.8.x.\
+We would recommend against using 1.8 or arguably any version below latest, due to the presence of security vulnerabilities or other exploits.\
+One of the main reasons for choosing to run a 1.8.x fork is for good performance on older hardware, or for pre-1.9 PvP.\
+See below for some 1.8.x forks that are actively maintained:
+
+<Cards>
+    <Card href="https://github.com/hpfxd/PandaSpigot" title = "PandaSpigot"></Card>
+    <Card href="https://github.com/Wind-Development/WindSpigot" title = "WindSpigot"></Card>
+    <Card href="https://github.com/CobbleSword/NachoSpigot" title = "NachoSpigot"></Card>
+</Cards>
+
+## Forge
+Forge is a modded server (and API) that allows for the use of mods on your server, which can add new content, mechanics and features to the game (Unlike Vanilla, Spigot, Paper, etc).\
+Unlike Paper, it contains **no optimizations** and is often very laggy when on larger modpacks.
+
+<Card href="https://files.minecraftforge.net/net/minecraftforge/forge/" title="Download">
+  Download a copy of Forge via [minecraftforge.net](https://files.minecraftforge.net/net/minecraftforge/forge/).
+</Card>
+
+## NeoForge
+NeoForge is a fork of Forge, maintained by most of the original Forge development team - excluding LexManos (see [this post](https://neoforged.net/news/theproject/)).\
+It aims to fix the issues surrounding the community and provide a better development experience for modders and more recent versions of Minecraft itself.
+
+<Callout type="info" title="Note">
+  Due to NeoForge being a relatively new server software, there is not as many mods that completely support it compared to the likes of Forge and Fabric.
+  However, many mods are slowly being updated to support NeoForge, and many popular modpacks exist for it.
+</Callout>
+
+<Card href="https://neoforged.net/" title="Download">
+  Download a copy of the latest NeoForge server software via [neoforged.net](https://neoforged.net/).
+</Card>
+
+## Fabric
+Similar to Forge and NeoForge, Fabric is a modded server software (and API). Like Forge & NeoForge, it allows you to add new content (such as mobs, biomes, generation etc) to the game with ease.\
+It is known for being more lightweight with a more extensive array of performance mods compared to Forge, such as Lithium, Moonrise, Starlight, etc.
+
+<Card href="https://fabricmc.net/use/installer/" title="Download">
+  Download a copy of the latest Fabric server software (installer) via [fabricmc.net](https://fabricmc.net/use/installer/).
+</Card>
+<br></br>
+
+# Proxy Servers
+Proxies are server softwares that allow you to connect multiple servers together, making it more seamless for switching between servers / gamemodes.\
```

Comment:
Another case of "softwares". Uncountable noun, should be "software" in the plural, etc etc.

Reply:
Resolved in [d20c33d](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/d20c33d060a328b2cce5bbeb24c70aee63e38311)